lib/setup/setupUpload.js

```diff
@@ -13,7 +14,7 @@ function Upload(options) {
     const objects     = options.objects;
     let mime;
     // attName = file.split('/' + tools.appName + '.');
-    const regApp = new RegExp('\/' + tools.appName + '\.', 'i');
+    const regApp = new RegExp('/' + tools.appName.replace(/\./g, '\\.') + '\\.', 'i');

```

Comment:
No need to escape / in the string. To get `\.` in the RegExp, we need `\\.` in the string.